### PR TITLE
Feature/add panel component

### PR DESCRIPTION
**Pull request by hdfedscottquested**

### CODE REVIEW

File: components/atoms/icon/__snapshots__/Icon.test.js.snap [hunk 1]
```diff
@@ -7,8 +7,9 @@ exports[`Check the icon renders it should match the snapshot 1`] = `
     Object {
       "backgroundColor": "#0085ff",
       "borderColor": undefined,
-      "height": 39,
-      "width": 39,
+      "borderWidth": 3,
+      "height": 60,
+      "width": 60,
```
Comment (hdfeddanjackson) [inline components/atoms/icon/__snapshots__/Icon.test.js.snap:12]: Remove comma from the end of line 12.

File: components/molecules/panel/panel.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			background-image: url('../../../src/img/panel-bottom-right.svg');
+			background-position: 100% 100%;
+		}
+	}
+
+	&--centered-content {
+		text-align: center;
+	}
+
+	&__header {
```
Comment (hdfeddanjackson) [inline components/molecules/panel/panel.scss:56]: could you not write the code like this:

```scss
&__header,
&__body {
	&--centered-content {
		text-align: center;
	}
}
```

**Review by hdfeddanjackson — APPROVED** [review]
Added a few comments, but will approve.

File: components/molecules/panel/panel.scss (hunk 2 above)
Comment (hdfedscottquested) [reply components/molecules/panel/panel.scss:56]: Yeah, good spot. I'll go one better and add the footer in there as well

Comment (hdfedscottquested) [reply components/molecules/panel/panel.scss]: Done

**Pull request merged**